x-pack/plugins/ml/public/application/routing/routes/new_job/index_or_search.tsx

```diff
@@ -72,6 +72,17 @@ const getLogCategorizationBreadcrumbs = (navigateToPath: NavigateToPath, basePat
   },
 ];
 
+const getChangePointDetectionBreadcrumbs = (navigateToPath: NavigateToPath, basePath: string) => [
+  getBreadcrumbWithUrlForApp('ML_BREADCRUMB', navigateToPath, basePath),
+  getBreadcrumbWithUrlForApp('AIOPS_BREADCRUMB_CHANGE_POINT_DETECTION', navigateToPath, basePath),
+  getBreadcrumbWithUrlForApp('CHANGE_POINT_DETECTION', navigateToPath, basePath),
+  {
+    text: i18n.translate('xpack.ml.aiopsBreadcrumbs.selectDateViewLabel', {
```

Comment:
Typo in the key here - should be `selectDataViewLabel`

Reply:
Fixed in https://github.com/elastic/kibana/pull/144093/commits/998cdd4777d337c22db72a311deacb070af587f8